style.css

```diff
@@ -0,0 +1,167 @@
+:root {
+  --color-primary: #1c1c1d;
+  --color-secondary: rgb(131, 128, 128);
+  --color-secondary-light: rgb(255, 254, 254);
+}
+body {
+  background-color: rgb(250, 245, 245);
+}
+.main {
+  width: 600px;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  padding: 25px;
+  background-color: white;
+  margin-top: 25px;
+  border-radius: 5px;
+  height: 500px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+.form {
+  padding: 25px;
+}
+.header {
+  margin-bottom: 25px;
+  font-weight: 600;
+  font-family: sans-serif Arial, Helvetica, sans-serif;
+  color: #1c1c1d;
+}
+.search {
+  width: 100%;
+  border: none;
+  outline: none;
+  border-radius: 0;
+  color: var(--color-secondary);
+}
+.search-box {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: solid 1px rgb(228, 222, 222);
+}
+
+button i {
+  color: var(--color-secondary);
+}
+button i:hover {
+  color: rgb(194, 186, 186);
+  transition: 0.7s;
+}
+.search-btn:hover {
+  border: solid 1px rgb(255, 254, 254);
+  background-color: var(--color-secondary-light);
+}
+.names-list {
+  margin-top: 30px;
+}
+.list-item-wrapper {
+  margin-bottom: 5px;
+  border-radius: 5px;
+}
+.list-item-wrapper:hover {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
+  transform: scale(1.01);
+  color: #1c1c1d;
+}
+.item-name {
+  color: var(--color-secondary);
+}
+
+.custom-checkbox input {
+  display: none;
+}
+
+.custom-checkbox span {
+  border: 1px solid #7e8a94;
+  float: right;
```

Comment:
Floating is rarely used these days. Consider rewriting the item using flexbox. It is a lot easier.
Float can cause container collapsing as it will go out from the regular flow.
More information: https://www.linkedin.com/learning/css-essential-training-3/float-and-collapsed-container?autoplay=true